docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -654,6 +654,7 @@ The perturbation process is as follows:
 **10%** of the time, replace the sampled person/household's PUMA with a PUMA selected at random according to the
 probabilities in the appropriate file below.
 (In the case of a household, all simulants in that household are assigned the same replacement PUMA.)
+In the case of an individual, only PUMAs with an existing non-GQ household in the simulation should be selected.
```

Comment:
Ah right. No, I don't think you could have thought of this earlier, because this issue is introduced in this PR. In the previous version, the additional perturbation was done at matching time so it didn't require the simulant to be selected.

Personally, I don't see an issue with the first approach, and it's equivalent to what I had written before.

I'll update this PR to handle this case.